src/main.js

```diff
@@ -194,16 +222,19 @@ async function createGradesOverviewTable(div) {
         let gradePercentageRounded = Math.round(10000 * _Student.gradesCount[gradeId] / _Student.totalGrades) / 100;
         gradesTableTemplate = gradesTableTemplate.replace('percentage-' + gradeId, gradePercentageRounded + "%");
     }
+    gradesTableTemplate = gradesTableTemplate.replace("QUANTITY", await i18n.getMessage("quantity"));
+    gradesTableTemplate = gradesTableTemplate.replace("DISTRIBUTION", await i18n.getMessage("distribution"));
     gradeOverviewTable.innerHTML = gradesTableTemplate;
     div.insertBefore(gradeOverviewTable, div.firstChild);
 }
 
 /*
  * Create a heading that displays the number of achieved credits.
  */
-function createTotalCreditsTitle(div) {
+async function createTotalCreditsTitle(div) {
```

Comment:
I you wouldn't load the JSON for every call of `i18n.getMessage()`, this method wouldn't need to be async.

Reply:
done